templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -190,5 +190,136 @@
 				</div>
 			{{end}}
 		{{end}}
+
+		{{if .IssueDependenciesEnabled}}
+		<div class="ui divider"></div>
+
+		<div class="ui depending">
+			<span class="text"><strong>{{.i18n.Tr "repo.issues.dependency.title"}}</strong></span>
+			<br>
+			{{if .BlockedByDependencies}}
+			<span class="text">
+				{{if .Issue.IsPull}}
+					{{.i18n.Tr "repo.issues.dependency.issue_closing_blockedby"}}:
+				{{else}}
+					{{.i18n.Tr "repo.issues.dependency.pr_closing_blockedby"}}:
+				{{end}}
+			</span>
+			<div class="ui relaxed divided list">
+				{{range .BlockedByDependencies}}
+				<div class="item">
+					<div class="right floated content">
+						<a class="delete-dependency-button" onclick="deleteDependencyModal({{.Issue.ID}}, 'blockedBy');">
+							<i class="delete icon text red"></i>
+						</a>
+						{{if .IsClosed}}
+						<div class="ui red mini label">
+							<i class="octicon octicon-issue-closed"></i>
+						</div>
+						{{else}}
+						<div class="ui green mini label">
+							<i class="octicon octicon-issue-opened"></i>
+						</div>
+						{{end}}
+					</div>
+					<div class="ui black label">#{{.Index}}</div>
+					<a class="title has-emoji" href="{{$.RepoLink}}/issues/{{.Index}}">{{.Title}}</a>
+				</div>
+				{{end}}
+			</div>
+			{{end}}
+
+			{{if .BlockingDependencies}}
+			<div class="ui relaxed divided list">
+				<span class="text">
+					{{if .Issue.IsPull}}
+						{{.i18n.Tr "repo.issues.dependency.pr_close_blocks"}}:
+					{{else}}
+						{{.i18n.Tr "repo.issues.dependency.issue_close_blocks"}}:
+					{{end}}
+				</span>
+				{{range .BlockingDependencies}}
+				<div class="item">
+					<div class="right floated content">
+						<a class="delete-dependency-button" onclick="deleteDependencyModal({{.Issue.ID}}, 'blocking');">
+							<i class="delete icon text red"></i>
+						</a>
+						{{if .IsClosed}}
+						<div class="ui red tiny label">
+							<i class="octicon octicon-issue-closed"></i>
+						</div>
+						{{else}}
+						<div class="ui green mini label">
+							<i class="octicon octicon-issue-opened"></i>
+						</div>
+						{{end}}
+					</div>
+					<div class="ui black label">#{{.Index}}</div>
+					<a class="title has-emoji" href="{{$.RepoLink}}/issues/{{.Index}}">{{.Title}}</a>
+				</div>
+				{{end}}
+			</div>
+			{{end}}
+
+			{{if (and (not .BlockedByDependencies) (not .BlockingDependencies))}}
+			<p>{{.i18n.Tr "repo.issues.dependency.no_dependencies"}}</p>
+			{{end}}
+			<div>
+				<button class="fluid green ui button" onclick="showAddDependencyModal();">
+					{{.i18n.Tr "repo.issues.dependency.add"}}
+				</button>
+			</div>
+		</div>
+	</div>
+</div>
+
+
+<div class="ui mini modal add-dependency">
```

Comment:
Instead of modal, you can use just one input with button on sidebar (https://semantic-ui.com/elements/input.html#action).

Reply:
Or use a drop-down selection list like labels does.

Reply:
> Or use a drop-down selection list like labels does.
Good luck finding your issue if you have > 1000 issues... :grin:

Reply:
It should/will be autocomplete list/input once issues api calls will be implemented

Reply:
For now i'm solving this via a semantic ui searchable dropdown with _all_ issues of the repository. (I know this is not the most performant solution, but it currently is the only one possible)

Reply:
@kolaente I think you can skip this problem for this PR, just use simple input where you have to write issue index by hand. You can make new PR later to add option to list issues.

Reply:
Well, when i saw your response, i've already added it...